Dockerfile

```diff
@@ -6,7 +6,8 @@
 # Should be kept in sync with jenkins.properties of pom.xml
 # Patch version is not to be considered, we prefer to base the image off the latest LTS of the line
 # and keep the dependency on the baseline in pom.xml
-FROM jenkins/jenkins-experimental:latest-jdk11
+# TODO: revert to jenkins/jenkins:jdk11 once 2.155 is released
```

Comment:
Could be done now?

Reply:
Nope. The release flow as borked again, waiting for the fix